airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -10,7 +10,8 @@
         "title": "Host",
         "description": "Hostname of the database.",
         "type": "string",
-        "order": 0
+        "order": 0,
```

Comment:
If the feature flag is off, starting the order from 0 for each group will shuffle the fields.

It seems like we can keep the current orders except for the JDBC-params / Replication method case in the "Advanced" section, maybe it's fine to keep them in the current order (just making sure that we can roll back with the feature flag)

Reply:
Good catch - I've updated this to undo the `order` changes I made to this spec. I also moved the `schemas` field into the `db` group since I think it belongs better there.

I think the advanced section is fine with the optional fields coming first, so we can leave that as is for now.